mysite/settings.py

```diff
@@ -122,4 +122,23 @@
 # https://docs.djangoproject.com/en/1.9/howto/static-files/
 
 STATIC_URL = '/static/'
-STATIC_ROOT = os.path.join(BASE_DIR, 'static')
\ No newline at end of file
+<<<<<<< HEAD
```

Comment:
Please learn about merge conflict markers and fix this module accordingly. (You have to pick what to keep in this module and what to wipe out).
Also please run `pre-commit install` to enable pre-commit tool do checks automatically in the local repo.